pyproject.toml

```diff
@@ -45,7 +45,7 @@ django-markdown-deux = "^1.0.6"
 django-mathfilters = "*"
 django-pghistory = "^2.9.0"
 django-ratelimit = "^4.1.0"
-django-storages = "^1.13.2"
+django-storages = "^1.14.2"
```

Comment:
Changelog: https://github.com/jschneier/django-storages/blob/master/CHANGELOG.rst

Reply:
Looks like there are a few tweaks we should make:

> Important: The namespace of this backend has changed from S3Boto3 to S3. There are no current plans to deprecate and remove the old namespace but please update if you can. All paths, imports, and classes that previously referred to s3boto are now s3. E.g S3Boto3Storage has been changed to S3Storage and S3Boto3StorageFile has been changed to S3File. (https://github.com/jschneier/django-storages/pull/1289). Additionally the install extra is now s3

Reply:
Updated with those tweaks.